test/system/randomwalk/conf/modules/MultiTable.xml

```diff
@@ -29,11 +29,11 @@
 
 <node id="dummy.ToAll">
   <edge id="mt.CreateTable" weight="20"/>
-  <edge id="mt.Write" weight="100"/>
+  <edge id="mt.Write" weight="10"/>
   <edge id="mt.CopyTable" weight="5"/>
+  <edge id="mt.BulkImport" weight="100"/>
   <edge id="mt.OfflineTable" weight="10"/>
   <edge id="mt.DropTable" weight="3"/>
-  <edge id="END" weight="1"/>
```

Comment:
Should this be kept?

Reply:
I don't think there is any benefit to having the test end when the job can be killed with the scripts.